### DISCUSSION TITLE
Maximum number of keepalive connections?

### DISCUSSION THREAD

**Discussion by ozgurkazancci** [ant-media/Ant-Media-Server#6222] [General]
Optimizing nginx for Reverse TLS proxy for owncast - I'd like to ask;

What's the maximum number of keepalive connections the Ant-Media-Server web server can maintain?

Thanks.

**Comment by burak-58** [comment]
Hi @ozgurkazancci,
Thank you for your question. Let me convert this to a github discussion so that community can follow the discussion.

**Comment by muratugureminoglu** [comment]
Hi @ozgurkazancci 

There are no restrictions on this. So it depends on your server. You can restrict it in red5.properties file.

https://github.com/ant-media/Ant-Media-Server/blob/master/src/main/server/conf/red5.properties#L71
